README.md

```diff
@@ -610,6 +613,21 @@ Usage: `nerdctl create [OPTIONS] IMAGE [COMMAND] [ARG...]`
 
 The `nerdctl create` command similar to `nerdctl run -d` except the container is never started. You can then use the `nerdctl start <container_id>` command to start the container at any point.
 
+### :whale: nerdctl cp
+Copy files/folders between a running container and the local filesystem
+
+Usage:
+- `nerdctl cp [OPTIONS] CONTAINER:SRC_PATH DEST_PATH|- [flas]`
+- `nerdctl cp [OPTIONS] SRC_PATH|- CONTAINER:DEST_PATH`
+
+Flags:
+- :whale: -L, `-c, --compress` Compress tar with gzip.
+- :whale: -L, `-L, --follow-link` Always follow symbol link in SRC_PATH.
+- :whale: -L, `--no-preserve` The copied file/directory's ownership and permissions will not be preserved in the container.
+- :whale: -L, `-q, --quite` Suppress progress output during copy. Progress output is automatically suppressed if no terminal is attached.
+- :whale: -L, `--retries` Set number of retries to complete a copy operation from a container. Specify 0 to disable or any negative value for infinite retrying. The default is 0 (no retry).
```

Comment:
not whale

Reply:
And why do you need to retry?

Reply:
if cp fails we retry copy from the last readed byte

Reply:
Why fails? Shouldn’t it be retried by the user?

Reply:
https://pkg.go.dev/io#Reader : looking at the `Read` comment, it may encounters an error after successfully reading n > 0 bytes. So the idea is automatically retry (n times) without user  without user intervention

it is a comfort for the user :D , WDYT ?

Reply:
Any syscall may fail, but we don’t have such auto retry feature for other commands.